Documentation/gettingstarted/encryption-ipsec.rst

```diff
@@ -346,9 +346,3 @@ Troubleshooting
         $ tc filter show dev eth0 ingress
         filter protocol all pref 1 bpf chain 0
         filter protocol all pref 1 bpf chain 0 handle 0x1 bpf_network.o:[from-network] direct-action not_in_hw id 1145 tag 51b408acf94aa23f jited
-
-Disabling Encryption
```

Comment:
Could you please open an issue so we can track this and add it back when the time is come?

Reply:
NAK from me. Lets create an issue and fix it. The documentation has been this way from day 1 also when this happens is a specific to kernel versions and modes so correctly describing the conditions would require rework.

Reply:
OK! I am removing the commit introducing the change. Do we still need to create an issue for tracking this or will #15993 be sufficient?